charts/observability-pipelines-worker/templates/configmap.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
     checksum/config: {{ tpl (toYaml .Values.config) . | sha256sum }}
 data:
   {{- if .Values.config }}
-  config.yaml: | {{ tpl (toYaml .Values.config) . | nindent 4 }}
+  pipelines.yaml: | {{ tpl (toYaml .Values.config) . | nindent 4 }}
   {{- else }}
 {{ fail "A `config` must be specified, see https://docs.datadoghq.com/observability_pipelines/reference for all options" }}
```

Comment:
The filename has become `pipelines.yaml`, do you still want the user to configure it's contents via a key `config`?

Reply:
Yes, good timing, I was about to comment here. I think `config` is too general, especially with bootstrap involved. Is there a good way to refactor this other than find/replace? I think `pipelinesConfig` (open to suggestions) may be better.

Reply:
Yeah, that name seems fine to me? I'm terrible with names though. Find and replace should do it.

We'll probably need to make a couple more changes to account for the `bootstrap.yaml` however.

Reply:
I've updated the naming to `pipelineConfig` in [31a5536](https://github.com/DataDog/helm-charts/pull/952/commits/31a55360b1dcf5d876ff1087c50c54bcca9950e1) and beyond.